--- damask-3.0.0-beta2-orig/src/CLI.f90	2024-06-01 22:57:34.241147368 +0200
+++ damask-3.0.0-beta2/src/CLI.f90	2024-06-01 22:58:08.680274328 +0200
@@ -123,7 +123,7 @@
 
   print'(/,1x,a)', 'Compiled with: '//compiler_version()
   print'(1x,a)',   'Compiled on: '//CMAKE_SYSTEM
-  print'(1x,a)',   'Compiler options: '//compiler_options()
+  !print'(1x,a)',   'Compiler options: '//compiler_options()
 
   ! https://github.com/jeffhammond/HPCInfo/blob/master/docs/Preprocessor-Macros.md
   print'(/,1x,a)', 'Compiled on: '//__DATE__//' at '//__TIME__
