--- damask-3.0.0-alpha7-orig/src/mesh/discretization_mesh.f90	2023-02-12 06:39:57.979772705 +0100
+++ damask-3.0.0-alpha7/src/mesh/discretization_mesh.f90	2023-02-12 06:40:36.692355064 +0100
@@ -56,11 +56,6 @@
   real(pReal), dimension(:,:,:), allocatable :: &
     mesh_ipCoordinates                                                                              !< IP x,y,z coordinates (after deformation!)
 
-  external :: &
-#ifdef PETSC_USE_64BIT_INDICES
-    DMDestroy, &
-#endif
-    DMView                                                                                          ! ToDo: write interface
   public :: &
     discretization_mesh_init, &
     mesh_FEM_build_ipVolumes, &
