--- damask-3.0.0-alpha8-orig/CMakeLists.txt	2023-11-14 20:35:34.175176930 +0100
+++ damask-3.0.0-alpha8/CMakeLists.txt	2023-11-14 20:36:16.854822947 +0100
@@ -111,6 +111,7 @@
 
 file(STRINGS "$ENV{PETSC_DIR}/$ENV{PETSC_ARCH}/lib/petsc/conf/petscvariables" PETSC_EXTERNAL_LIB REGEX "PETSC_EXTERNAL_LIB_BASIC = .*$?")
 string(REPLACE "PETSC_EXTERNAL_LIB_BASIC = " "" PETSC_EXTERNAL_LIB "${PETSC_EXTERNAL_LIB}")
+string(REPLACE "-lhdf5_hl -lhdf5" "-lhdf5hl_fortran -lhdf5_fortran -lhdf5_hl -lhdf5" PETSC_EXTERNAL_LIB "${PETSC_EXTERNAL_LIB}")
 message("PETSC_EXTERNAL_LIB:\n${PETSC_EXTERNAL_LIB}\n")
 
 file(STRINGS "$ENV{PETSC_DIR}/$ENV{PETSC_ARCH}/lib/petsc/conf/petscvariables" PETSC_INCLUDES REGEX "PETSC_FC_INCLUDES = .*$?")
