diff --color --unified --recursive --text damask-3.0.0-alpha6-orig/cmake/Compiler-GNU.cmake damask-3.0.0-alpha6/cmake/Compiler-GNU.cmake
--- damask-3.0.0-alpha6-orig/cmake/Compiler-GNU.cmake	2022-02-19 08:33:24.893673992 +0100
+++ damask-3.0.0-alpha6/cmake/Compiler-GNU.cmake	2022-02-19 08:36:52.980247691 +0100
@@ -19,7 +19,7 @@
   set (OPTIMIZATION_FLAGS "-O3 -march=native -funroll-loops -ftree-vectorize -flto")
 endif ()
 
-set (STANDARD_CHECK "-std=f2018 -pedantic-errors" )
+set (STANDARD_CHECK "-std=f2018" )
 
 #------------------------------------------------------------------------------------------------
 # Fine tuning compilation options
@@ -101,11 +101,13 @@
 set (COMPILE_FLAGS "${COMPILE_FLAGS} -Wsuggest-attribute=pure")
 set (COMPILE_FLAGS "${COMPILE_FLAGS} -Wsuggest-attribute=noreturn")
 set (COMPILE_FLAGS "${COMPILE_FLAGS} -Wconversion-extra")
-set (COMPILE_FLAGS "${COMPILE_FLAGS} -Wimplicit-procedure")
 set (COMPILE_FLAGS "${COMPILE_FLAGS} -Wno-unused-parameter")
 set (COMPILE_FLAGS "${COMPILE_FLAGS} -ffpe-summary=all")
 # print summary of floating point exeptions (invalid,zero,overflow,underflow,inexact,denormal)
 
+set (COMPILE_FLAGS "${COMPILE_FLAGS} -fallow-argument-mismatch")
+# needed for MPICH/gfortran 10 (MPI_Bcast)
+
 # Additional options
 # -Warray-temporarieswarnings:   because we have many temporary arrays (performance issue?)
 # -Wimplicit-interface:          no interfaces for lapack/MPI routines
