--- damask-3.0.0-alpha7-orig/cmake/Compiler-GNU.cmake	2022-10-10 13:58:08.004855203 +0200
+++ damask-3.0.0-alpha7/cmake/Compiler-GNU.cmake	2022-10-10 13:59:02.331470037 +0200
@@ -19,7 +19,7 @@
   set (OPTIMIZATION_FLAGS "-O3 -march=native -funroll-loops -ftree-vectorize -flto")
 endif ()
 
-set (STANDARD_CHECK "-std=f2018 -pedantic-errors" )
+set (STANDARD_CHECK "-std=f2018" )
 
 #------------------------------------------------------------------------------------------------
 # Fine tuning compilation options
@@ -101,7 +101,7 @@
 set (COMPILE_FLAGS "${COMPILE_FLAGS} -Wsuggest-attribute=pure")
 set (COMPILE_FLAGS "${COMPILE_FLAGS} -Wsuggest-attribute=noreturn")
 set (COMPILE_FLAGS "${COMPILE_FLAGS} -Wconversion-extra")
-set (COMPILE_FLAGS "${COMPILE_FLAGS} -Wimplicit-procedure")
+set (COMPILE_FLAGS "${COMPILE_FLAGS} -fallow-argument-mismatch")
 set (COMPILE_FLAGS "${COMPILE_FLAGS} -Wunused-parameter")
 set (COMPILE_FLAGS "${COMPILE_FLAGS} -ffpe-summary=all")
 # print summary of floating point exeptions (invalid,zero,overflow,underflow,inexact,denormal)
