Only in damask-3.0.0-alpha7-orig: damask-3.0.0-alpha7
diff --color -urt damask-3.0.0-alpha7-orig/src/CLI.f90 damask-3.0.0-alpha7/src/CLI.f90
--- damask-3.0.0-alpha7-orig/src/CLI.f90	2023-07-13 02:31:37.131002312 +0200
+++ damask-3.0.0-alpha7/src/CLI.f90	2023-07-13 03:33:12.799603660 +0200
@@ -21,7 +21,7 @@
 !--------------------------------------------------------------------------------------------------
 #define PETSC_MAJOR 3
 #define PETSC_MINOR_MIN 12
-#define PETSC_MINOR_MAX 18
+#define PETSC_MINOR_MAX 20
 
 module CLI
   use, intrinsic :: ISO_fortran_env
