diff --color -urt damask-3.0.0-alpha7-orig/CMakeLists.txt damask-3.0.0-alpha7/CMakeLists.txt
--- damask-3.0.0-alpha7-orig/CMakeLists.txt	2023-07-13 02:31:37.141002687 +0200
+++ damask-3.0.0-alpha7/CMakeLists.txt	2023-07-13 03:32:16.493928471 +0200
@@ -11,7 +11,7 @@
 project(Prerequisites LANGUAGES)
 set(ENV{PKG_CONFIG_PATH} "$ENV{PETSC_DIR}/$ENV{PETSC_ARCH}/lib/pkgconfig:$ENV{PKG_CONFIG_PATH}")
 pkg_check_modules(PETSC_MIN REQUIRED PETSc>=3.12.0 QUIET) #CMake does not support version range
-pkg_check_modules(PETSC REQUIRED PETSc<3.19.0)
+pkg_check_modules(PETSC REQUIRED PETSc<3.21.0)
 
 pkg_get_variable(CMAKE_Fortran_COMPILER PETSc fcompiler)
 pkg_get_variable(CMAKE_C_COMPILER PETSc ccompiler)
Only in damask-3.0.0-alpha7-orig: damask-3.0.0-alpha7
