--- damask-3.0.0-alpha7-orig/python/setup.cfg	2022-10-10 11:34:22.934631052 +0200
+++ damask-3.0.0-alpha7/python/setup.cfg	2022-10-10 11:54:32.197194977 +0200
@@ -6,7 +6,7 @@
 url = https://damask.mpie.de
 description = DAMASK processing tools
 long_description = Pre- and post-processing tools for DAMASK
-license: AGPL3
+license = AGPL3
 classifiers =
     Intended Audience :: Science/Research
     Topic :: Scientific/Engineering
@@ -17,14 +17,12 @@
 [options]
 packages = find:
 include_package_data = true
-python_requires = >= 3.8
+python_requires = >= 3.9
 install_requires =
-    importlib-metadata; python_version<"3.8"
-    pandas; python_version<="0.24"                                                                  # requires numpy
-    numpy; python_version<="1.17"                                                                   # needed for default_rng
-    scipy; python_version<="1.2"
-    h5py; python_version<="2.9"                                                                     # requires numpy
-    vtk; python_version<="8.1"
-    matplotlib; python_version<="3.0"                                                               # requires numpy, pillow
-    pyyaml; python_version<="3.12"
-setup_requires = setuptools
+    pandas>=0.24                                                                                    # requires numpy
+    numpy>=1.17                                                                                     # needed for default_rng
+    scipy>=1.2
+    h5py>=2.9                                                                                       # requires numpy
+    vtk>=8.1
+    matplotlib>=3.0                                                                                 # requires numpy, pillow
+    pyyaml>=3.12
